### PR TITLE
Added Osteolytica project wrapup blog post

### PR DESCRIPTION
**Pull request by twinkarma**

### CODE REVIEW

**Comment by willfurnass** [comment]
@twinkarma CI could not build this:
```
..[2018-09-17T16:39:25Z] ERROR: Nikola: Two posts are trying to generate blog/2nd-gpu-seminar-amr/index.html: posts/2017_05_30_gpu-computing-seminar-adaptive-mesh-refinement.md and posts/2018_09_17_osteolytica-project-wrapup.md
[2018-09-17T16:39:25Z] ERROR: Nikola: Two posts are trying to generate blog/2nd-gpu-seminar-amr/index.md: posts/2017_05_30_gpu-computing-seminar-adaptive-mesh-refinement.md and posts/2018_09_17_osteolytica-project-wrapup.md
```

**Comment by twinkarma** [comment]
I've fixed the issue now and have also added current & previous DL training events.

File: site/posts/2018_09_17_osteolytica-project-wrapup.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Time series scans from longitudinal studies
+
+The last new development in Osteolytica is to add an experimental feature to allow comparison of the same set of bone over time. This happens in longitudinal studies that tracks the progression of cancer in response to the administration of a new drug for example.
+
+While theoretically it should be possible to compare the bone volumes over time period directly, there’s a problem where bone volumes in each scan do not align to each other exactly. The alignment problem was solved by using a local optimisation algorithm (COBYLA provided in nlopt) in order to maximise the overlap between the two volumes in the time series. The initial result of this alignment process is shown in **Figure 5** for two volumes scanned at week 0 and week 3. The white colour shows where the volumes overlap, blue and red colours represent the week 0 sample and week 3 samples respectively that did not overlap. **Figure 5a** shows the two volumes before alignment and **Figure 5b** shows the volumes after alignment. In this case it can be seen that there has been bone growth after the administering of drugs and that lesions are starting to be filled in.
```
Comment (willfurnass) [inline site/posts/2018_09_17_osteolytica-project-wrapup.md]: Add links to 'COBYLA' and 'nlopt'?

File: site/posts/2018_09_17_osteolytica-project-wrapup.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## The reconstruction process
+The goal of Osteolytica is to determine the surface area missing from the bone as a result of osteolytic lesions without needing to refer to prior samples. In order to do this, Osteolytica performs a bone surface reconstruction in two stages, expansion, and contraction.
+
+For the expansion stage (**Figure 1**), volumetric expansion is performed on the bone volume sample. This expansion closes the holes within the sample, generating a water-tight volume that also allows the empty areas inside and outside of the volume to be differentiated.
+
+![Expansion process](/images/2018_09_17_osteolytica-project-wrapup/osteolytica-expansion.jpg)
+**Figure 1:** The expansion process shown in 2D. a) Bone volume is shown in black. b) The volume is expanded (red) until all the gaps are enclosed. c) Once the expansion is completed, the inside volume can be filled in and the contraction starts from the expansion border (blue line)
```
Comment (willfurnass) [inline site/posts/2018_09_17_osteolytica-project-wrapup.md:31]: On my display figure captions start to the R of the image(s) then wrap around beneath them.  Better to have a blank link in the markdown between the image and caption so the caption is treated as a separate  paragraph?

File: site/posts/2018_09_17_osteolytica-project-wrapup.md (hunk 1 above)
Comment (willfurnass) [inline site/posts/2018_09_17_osteolytica-project-wrapup.md]: Any code publicly available?  Want to add anything re the experience of working with academics / designing the UI?

**Review by willfurnass — CHANGES_REQUESTED** [review]

**Comment by willfurnass** [comment]
Also, in future it's better to submit multiple PRs for e.g. new blog post and new training info.  Makes the review process simpler.

File: site/posts/2018_09_17_osteolytica-project-wrapup.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<!--
+.. title: Osteolytica Project Wrapup
+.. author: Twin Karmakharm
+.. slug: osteolytica-project-wrapup
+.. date: 2018-09-17 14:00:00 UTC+01:00
```
Comment (willfurnass) [inline site/posts/2018_09_17_osteolytica-project-wrapup.md]: Bump date so appears as most recent blog post.

File: site/posts/2018_09_17_osteolytica-project-wrapup.md (hunk 1 above)
Comment (twinkarma) [reply site/posts/2018_09_17_osteolytica-project-wrapup.md]: Added link in revision

File: site/posts/2018_09_17_osteolytica-project-wrapup.md (hunk 2 above)
Comment (twinkarma) [reply site/posts/2018_09_17_osteolytica-project-wrapup.md:31]: Fixed in revision

File: site/posts/2018_09_17_osteolytica-project-wrapup.md (hunk 1 above)
Comment (twinkarma) [reply site/posts/2018_09_17_osteolytica-project-wrapup.md]: No publicly available code as it's meant to be a commercial product in the future. I'll do another post  once we have the clinical data and we get to test the program on more samples.

File: site/posts/2018_09_17_osteolytica-project-wrapup.md (hunk 3 above)
Comment (twinkarma) [reply site/posts/2018_09_17_osteolytica-project-wrapup.md]: Done

**Comment by twinkarma** [comment]
Sorry about the stacked PRs, I'll make sure to put the separate changes on different branches next time.

**Pull request merged**